helpdesk_motive/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "Helpdesk Motive",
-    "version": "12.0.1.0.0",
+    "version": "12.0.1.0.1",
```

Comment:
@cesarlr addon version shouldn't be changed, OcaBot will do it

Reply:
Ok! I've just removed the change